recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure we are in the right place
+cd $SRC_DIR
+
+export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
```

Comment:
To hang on to existing `$LDFLAGS` content.

```suggestion
export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
```

Reply:
Fixed.